src/ratelimit.ts

```diff
@@ -16,54 +16,65 @@ export const RateLimiter = (
     const { ip } = ctx.request;
     const timestamp = Date.now();
 
+    const GET = await opt.store.get(ip)!
+    const SET = async (ip: string, options: Ratelimit) => {
+      await opt.store.set(ip, {
+        remaining: options.remaining,
+        lastRequestTimestamp: options.lastRequestTimestamp,
+      });
+    }
+    const DELETE = await opt.store.delete(ip)
+    const HAS = await opt.store.has(ip)
```

Comment:
`DELETE` is a method implementation, not a value that is being referred to somewhere. This shouldn't be its own value, instead, be where a method needs to be used.

`HAS` should be called in camel case, not capital letters, and should describe what it does. i.e. `existsInStore`.